src/main/java/org/springframework/cloud/deployer/spi/kubernetes/AbstractKubernetesDeployer.java

```diff
@@ -156,52 +149,61 @@ protected AppStatus buildAppStatus(String id, PodList podList, ServiceList servi
 		return statusBuilder.build();
 	}
 
+	void logPossibleDownloadResourceMessage(Resource resource) {
```

Comment:
This method can be set to protected. Seems like it is only used in child classes.